tests/cli.rs

```diff
@@ -279,8 +279,8 @@ fn test_write_all_values(#[case] number: i32, #[case] expected_output: &str) ->
          }
         }
     "#,
-    &["nested_type: NestedType1"],
-    &["pub fn nested_type(&self) -> &NestedType1 {"]
+    &["nested_type: i64"],
```

Comment:
(🗺️ PR tour) This file contains changes for having nested sequence type/scalar type in the same struct/class without generating separate structure.